modules/proc_use-quay-build-dockerfiles.adoc

```diff
@@ -167,22 +167,25 @@ endif::[]
 
 .Procedure
 
-. On the *Builds* page, click *Start New Build*. Alternatively, you can click the `+` icon to reveal a drop-down menu. 
+. On the *Builds* page, click *Start New Build*.
 
-. When prompted, click *Select File* to upload a Dockerfile or an archive (for example, a `.tar.gz` or `.zip` file) that contains a Dockerfile at the root directory.
+. When prompted, click *Upload Dockerfile* to upload a Dockerfile or an archive (for example, a `.tar.gz` or `.zip` file) that contains a Dockerfile at the root directory.
```

Comment:
> an archive (for example, a `.tar.gz` or `.zip` file)
The new UI currently doesn't support this. We're debating how much this is actually used.